nautobot/docs/additional-features/job-scheduling-and-approvals.md

```diff
@@ -29,6 +31,10 @@ http://nautobot/api/extras/jobs/$JOB_ID/run/ \
 --data '{"schedule": {"name": "test", "interval": "future", "start_time": "2030-01-01T01:00:00.000Z"}}'
 ```
 
+For custom interval, a `crontab` parameter must be added.
+
+`--data '{"schedule": {"name": "test", "interval": "custom", "start_time": "2030-01-01T01:00:00.000Z", "crontab": "*/15 * * * *"}}'`
```

Comment:
This seems like a bit of a mixed case here. If there's a `crontab` why do we need to provide a `start_time`? Wouldn't the crontab determine the start time? Perhaps similar to "immediately" start times, perhaps we should consider using `datetime.now()` internally (See: `nautobot.extras.api.views._create_schedule()`)?

Reply:
They both should exist.

> Starting `start_time = time() + 1 week`, run this task every 15 minutes.

Reply:
i can have `start_time` be optional. when not specified, start immediately. i'm imagining cases where you'd want a job to run every 15 minutes, but you put that together on a friday and want it to start monday morning or something.

Reply:
Yeah, this is in line with the existing functionality where start time can act as a deferral on the start of a continual schedule.

Reply:
> i can have `start_time` be optional. when not specified, start immediately. i'm imagining cases where you'd want a job to run every 15 minutes, but you put that together on a friday and want it to start monday morning or something.

That's fair. In that case, please follow yours and Bryan's wisdom and disregard what we discussed in private.